remotedeploy.sh

```
@@ -10,6 +10,10 @@ if [ "$1" = "-d" ]; then
     sc64deployer $REMOTE debug --no-writeback
 fi
 
-if [ "$1" = "-du" ]; then
+if [ "$1" = "-dr" ]; then
+    sc64deployer $REMOTE debug --no-writeback --init "reboot"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)